src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -107,7 +107,7 @@ vec3 BRDF_Specular_GGX_Environment( const in GeometricContext geometry, vec3 spe
 
 	vec2 AB = vec2( -1.04, 1.04 ) * a004 + r.zw;
 
-	return specularColor * AB.x + AB.y;
+	return ( specularColor * AB.x + AB.y ) * grazingColor;
```

Comment:
We do need to have an unoptimized version of the environment BRDF.  This formulation is correct at the bounds of the metalness function, whether it is correct within the middle is harder to known as I am not sure the answer is clear.